### PR TITLE
Owl 2 DL Vocabulary

### PR DESCRIPTION
**Pull request by frederich-stine**
## Issue

This Pull Request resolves all requirements of Issue #435 .  *(Edited by AJN.  Originally referenced #406, which needed to be used as a baseline development state.)*

# Requirement 1

Datatype 2/3s incompletion resolved with owl:equivalentClass for all relevent vocabulary members.

# Requirement 2

Redundant definition subClass of removed for all relevent vocabulary members.

Review steps taken:

- [x] Pull request is against correct branch
- [x] CI passes in (CASE/UCO) feature branch
- [x] CI passes in (CASE/UCO) current unstable branch ([49ffd78](https://github.com/ucoProject/UCO-Archive/commit/49ffd788c313d7758b8cc7ba1e89f0b656f97fe8)) *(Note: [One extra commit](https://github.com/ucoProject/UCO-Archive/commit/e9deaac3f99d3c787b196c350ed1c0ae0c962a21) needed due to interaction with [another currently in-flight proposal](https://github.com/ucoProject/UCO/issues/251))*
- [ ] Impact on SHACL validation [reviewed](#TODO-commit) for CASE-Examples
- [ ] Impact on SHACL validation [remediated](#TODO-commit) for CASE-Examples 
- [ ] Impact on SHACL validation [reviewed](#TODO-commit) for casework.github.io
- [ ] Impact on SHACL validation [remediated](#TODO-commit) for casework.github.io 
- [ ] Solutions Approval vote logged on corresponding Issue

### CODE REVIEW

File: ontology/uco/vocabulary/vocabulary.ttl [hunk 1]
```diff
@@ -11,9 +11,15 @@
...
 	a rdfs:Datatype ;
-	rdfs:subClassOf rdfs:Resource ;
 	rdfs:label "Account Type Vocabulary"@en-US ;
-	owl:oneOf vocabulary:AccountTypeVocab-members ;
+	owl:equivalentClass [
+		a rdfs:Datatype ;
+		owl:onDatatype xsd:string ;
+		owl:withRestrictions (
+			owl:oneOf
+			vocabulary:AccountTypeVocab-members
```
Comment (ajnelson-nist) [inline ontology/uco/vocabulary/vocabulary.ttl:20]: Ah, @frederich-stine , I apologize I apparently misspoke when I described this to you.

The syntax change should start from the Feature-Issue-406 branch, not start from `develop` and merge into the 406 branch.  There is a correction in the 406 branch that you will not want to spend time repeating.

**Review by ajnelson-nist — CHANGES_REQUESTED** [review]

**Comment by frederich-stine** [comment]
@ajelson-nist 
Up to date with Feature-Branch-435, which is up to date with 406. Failure on make check due to placeholder test for OWL SHACL compatibility.

Should be good to merge, set target to 'develop'.

File: ontology/uco/vocabulary/vocabulary.ttl (hunk 1 above)
Comment (ajnelson-nist) [reply ontology/uco/vocabulary/vocabulary.ttl]: First, you're caught up with the test branch.  Thanks for the merge!

But, there's another syntax error here.  The OWL-to-RDF mapping doesn't quite make some constraints' separations clear, but there is a demonstration in the OWL Primer.  The short of it is, `:withRestrictions` should not be used to wrap `owl:oneOf`.  `owl:Restriction`s are for constraining literals in non-enumerative manners, like requiring integers be greater than a certain value, or strings match a regular expression.

You can see this demonstrated in the Primer with ["Show Turtle Syntax" turned on](https://www.w3.org/TR/owl2-primer/#OWL_Syntaxes).  Then search for `:withRestrictions` and `owl:oneOf` (esp. `:todderAge`).

If you just cut the `owl:withRestrictions (` and closing `)`, `make check` will create a re-normalized file for you and save you a lot of `\t` deletions.

Comment (frederich-stine) [reply ontology/uco/vocabulary/vocabulary.ttl]: Makes sense!
Have removed the `owl:withRestrictions` from the enumerated values. Tests passing again.

**Comment by frederich-stine** [comment]
> @ajelson-nist Up to date with Feature-Branch-435, which is up to date with 406. Failure on make check due to placeholder test for OWL SHACL compatibility.
> 
> Should be good to merge, set target to 'develop'.

Have fixed testing failure, good to merge now.

File: ontology/uco/vocabulary/vocabulary.ttl (hunk 1 above)
Comment (ajnelson-nist) [reply ontology/uco/vocabulary/vocabulary.ttl]: This thread's settled.  Thanks again, @frederich-stine !

**Comment by ajnelson-nist** [comment]
@eoghanscasey and @plbt5 - CASE PRs 99 through 103 catch CASE up with some of the practices adopted in UCO 0.8.0 through recent OCs' Solution Approvals.  Whomever sees them first, can you please merge them?

I realized CASE had fallen behind when I started reviewing the CASE Examples for this PR, and the new OWL SHACL check for `owl:oneOf` being in a blank node triggered on CASE's investigation-form vocabulary in *every* SHACL validation file.

This actually ends up being a behavior similar to `pyshacl`'s `--metashacl` flag, which reviews the SHACL graph with SHACL-SHACL before reviewing data.  `pyshacl` reviews the "Ontology" graph as well as reviewing the data graph, which is a tool behavior nuance (acknowledged every few months on the [pySHACL tracker](https://github.com/RDFLib/pySHACL/issues)) I'd either missed or forgotten.  Thus, if there's an error in the OWL or RDFS passed in to `pyshacl`'s `--ont-graph`, it will be reported alongside data errors.  This isn't necessarily something we have control over, except possibly adding a "Disable-OWL-review" switch to `case_validate` that turns off the OWL shapes by adding `sh:deactivated true` triples.

Paul, would you be interested in reviewing UCO's [`owl.ttl`](https://github.com/ucoProject/UCO/blob/develop/ontology/owl/owl.ttl) for shapes that would probably be appropriate to disable when using SHACL on the "ABox" side of a knowledge graph?  We can talk in more detail about this Monday.

**Comment by ajnelson-nist** [comment]
This was merged prematurely.

@eoghanscasey , please do not revert, that makes a tricky Git situation.  We will revert if the vote does not pass on Thursday.

**Pull request merged**